dashboard/modules/event/event_head.py

```diff
@@ -56,10 +64,27 @@ async def ReportEvents(self, request, context):
         received_events = []
         if request.event_strings:
             received_events.extend(parse_event_strings(request.event_strings))
-        logger.info("Received %d events", len(received_events))
+        logger.debug("Received %d events", len(received_events))
         self._update_events(received_events)
+        self.total_report_events_count += 1
+        self.total_events_received += len(received_events)
         return event_pb2.ReportEventsReply(send_success=True)
 
+    @dashboard_utils.async_loop_forever(10)
+    async def _periodic_state_print(self):
```

Comment:
This seems excessive for logging. We expect like maybe a few events a minute at most right? I think we should remove the periodic state print here and elsewhere.

Reply:
I will remove the periodic printing and find a better way to query this. Will keep the data/logic here (just no periodic printing)